### DISCUSSION TITLE
[Task]: Implementation of newer SOTA unsupervised anomaly detection methods

### DISCUSSION THREAD

**Discussion by phcarval** [open-edge-platform/anomalib#1121] [Feature Requests]
### What is the motivation for this task?

Hello,

I am currently working on comparing the performance of state-of-the-art unsupervised anomaly detection methods. Some of them, such as DSR (Zavrtanik et al. 2022) or ReConPatch (Hyun et al. 2023) are not implemented yet on Anomalib.

I wanted to know if there are currently any plans to implement these methods on Anomalib. If this is not the case, I would like to implement and add them to the library.

Thank you.

### Describe the solution you'd like

New methods : DSR, ReConPatch, PNI (Bae et al. 2022), MemSeg (Yang et al. 2022)

**Comment by samet-akcay** [comment]
Hi @phcarval, thanks for your interest. We have been wanting to add some of these models but unfortunately are occupied by other tasks. If you would like to implement these models, you are more than welcome! It would be good to have you on board as a contributor!

**Comment by phcarval** [comment]
Thank you for the answer! We will get to work on implemeting them then. We might also add the MVTec LOCO dataset which will likely be useful to us as well.

**Comment by samet-akcay** [comment]
Sounds great, thanks!

In regards to MVTec LOCO, there was a PR #538 for this, which was closed due to inactivity. @jpcbertoldo, do you have any plans to add loco. It would be good to check to avoid duplicated work.

**Reply by jpcbertoldo** [reply]
I stopped this at some point because there were some refactors coming up then I lost track of it. 
@phcarval if you want to make it go ahead : )
